content/en/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -83,9 +83,8 @@ In the `.yaml` file for the Kubernetes object you want to create, you'll need to
 
 The precise format of the object `spec` is different for every Kubernetes object, and contains nested fields specific to that object. The [Kubernetes API Reference](https://kubernetes.io/docs/reference/kubernetes-api/) can help you find the spec format for all of the objects you can create using Kubernetes.
 
-For example, the reference for Pod details the [`spec` field](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
-for a Pod in the API, and the reference for Deployment details the [`spec` field](/docs/reference/kubernetes-api/workload-resources/deployment-v1/#DeploymentSpec) for Deployments.
-In those API reference pages you'll see mention of PodSpec and DeploymentSpec. These names are implementation details of the Golang code that Kubernetes uses to implement its API.
+For example, see [this `spec` field](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec) reference for Pod details, for a Pod in the API, and [this `spec` field](/docs/reference/kubernetes-api/workload-resources/deployment-v1/#DeploymentSpec) reference for Deployment details, for Deployments.
```

Comment:
How about this:

```suggestion
For example, see the [`spec` field](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
in the reference for Pod. For each Pod, the `.spec` field specifies the pod and its desired state
(such as the container image name for each container within that pod).

Another example of an object specification is the
[`spec` field](/docs/reference/kubernetes-api/workload-resources/stateful-set-v1/#StatefulSetSpec)
for the StatefulSet API. For StatefulSet, the `.spec` field specifies the StatefulSet and
its desired state.
Within the `.spec` of a StatefulSet is a [template](/docs/concepts/workloads/pods/#pod-templates)
for Pod objects. That template describes Pods that the StatefulSet controller will create in order to
satisfy the StatefulSet specification.

Different kinds of object can also have different `.status`; again, the API reference pages
detail the structure of that `.status` field, and its content for each different type of object.
```

I switched to StatefulSet to avoid needing to explain what a ReplicaSet is.

Reply:
* I'd reword `in the reference for Pod.` (in the reference of what?)
* Next line (line no. 87) needs to be updated since we're now talking about `StatefulSet` instead of Deployment.

Reply:
Please feel free to revise your PR in line with the suggestions I've made, @alghe-global

Reply:
As for “see the `spec` field in the reference for Pod” - that makes sense to me. I've elided _documentation_; a longer form would be “see the `spec` field in the reference documentation for Pod”.

If you want to suggest other wording, that's also fine.